anitya/tests/db/test_models.py

```diff
@@ -155,6 +162,12 @@ def test_project_search(self):
         projects = models.Project.search(self.session, '*', page='asd')
         self.assertEqual(len(projects), 3)
 
+        projects = models.Project.search(self.session, '')
```

Comment:
I think you can break both of these new tests out into separate methods and add doc-blocks to them describing what they're testing. Tests should have as few assertions in them as possible so that when they fail, it's very quickly obvious what went wrong (ideally from the name of the test and doc-block).

Reply:
Yes, I should, but I didn't wanted to rewrite whole file. I used the same format that was already there.
But it is maybe good time to start with refactoring.

Reply:
Yeah, I don't think you need to break out the existing tests, but it's best not to make a bad situation worse, so I'd say any new assertions you're making should be in a new test case.